lib/std/zig/system.zig

```diff
@@ -110,6 +110,11 @@ pub const NativePaths = struct {
             try self.addLibDirFmt("/usr/lib{}", .{qual});
             try self.addLibDir("/usr/lib");
 
+            // Detect any native framework directories on MacOS
+            try self.addIncludeDir("/System/Library/Frameworks");
+            try self.addLibDirFmt("/System/Library/Frameworks{}", .{qual});
```

Comment:
`/System/Library/Frameworks32` and `/System/Library/Frameworks64`?
A quick google search shows no results for either of them.

Reply:
I don't think this is correct especially on Catalina or 10.15 which explicitly scraps all 32bit app support. Also, I'm not sure this will be enough to make things work on Big Sur where there is no `/System/Library/Frameworks` lying around anymore. Instead, one has to jump through the additional hoop of getting the fully qualified path to the SDK syslibroot, e.g., `-syslibroot $(xcrun --show-sdk-path)`.